app/controllers/users_controller.rb

```diff
@@ -118,8 +118,13 @@ def edit
 
   # PUT /facilities/:facility_id/users/:id
   def update
+    previous = @user.price_groups.to_a
     @user_form = user_form_class.new(@user)
     if @user_form.update_attributes(edit_user_params) && @user.update_price_group(price_group_params)
+      if previous != @user.price_groups.reload.to_a
+        new_price_group = @user.price_groups.to_a - previous
+        LogEvent.log(@user, :internal_changed, current_user, metadata: {price_group_rate: new_price_group.first.name})
```

Comment:
I think our normal style is to use spaces inside the hash braces. Not a big deal, though.

```suggestion
        LogEvent.log(@user, :internal_changed, current_user, metadata: { price_group_rate: new_price_group.first.name })
```